edb/common/markup/renderers/terminal.py

```diff
@@ -429,6 +435,18 @@ def _render_lang_String(self, element):
         self.buffer.write(
             xrepr(element.str, max_len=120), style=self.styles.literal)
 
+    def _render_lang_MultilineString(self, element):
+        with self.buffer.non_foldable_lines():
+            for line in element.str.splitlines():
+                self.buffer.new_line()
+                self.buffer.write(
+                    line,
+                    style=self.styles.literal
+                )
+            self.buffer.data.append((DEDENT_NO_NL, ))
+            self.buffer.new_line()
+            self.buffer.data.append((INDENT_NO_NL, ))
```

Comment:
It's been long time we've improved anything in this file...

Reply:
It's a rather nasty hack, but I couldn't find any prettier way to do this:

```
Delta(
   Query(
       multiline query text
       goes here
   )
)
```